checks/etherbox

```diff
@@ -220,3 +220,68 @@ check_info["etherbox.smoke"] = {
     "snmp_scan_function": etherbox_scan,
     "snmp_info": etherbox_info,
 }
+
+# .
+#   .--voltage-------------------------------------------------------------.
+#   |                             _ _                                      |
+#   |                 __   _____ | | |_ __ _  __ _  ___                    |
+#   |                 \ \ / / _ \| | __/ _` |/ _` |/ _ \                   |
+#   |                  \ V / (_) | | || (_| | (_| |  __/                   |
+#   |                   \_/ \___/|_|\__\__,_|\__, |\___|                   |
+#   |                                        |___/                         |
+#   '----------------------------------------------------------------------'
+
+
+def check_etherbox_voltage(item, no_params, info):
+    try:
+        name, value = etherbox_get_sensor(item, "5", info)
+    except Exception as error:
+        return 3, str(error)
+
+    state = 0
+    perfdata = [("voltage", value)]
+    extra_info = " - Status: OK"
+    if value != "0":
```

Comment:
I'm currently looking to implement those check in a migrated version of this check. Should there really be directly an alarm if voltage is unequal to zero? I would assume that this is better with configurable levels.

Reply:
I must take a look at the old plugin then i can answer this :)

Reply:
I used the same procedure as for smoke :)
The problem is that also for smoke there should be configurable values.
Inside the docs you find as smoke values something like "lower thresholds 0 / 0 , upper thresholds 90 / 110".
Now the problem, for the voltage sensor there is no documentation available.

Reply:
So we make this configurable and use 0 as the default, for smoke as well. Behavior is not changed but can be adopted later.